### ISSUE TITLE
Fix h1 in Hero section on mobile

### ISSUE THREAD

**Issue by JVSONV** [Vets-Who-Code/vets-who-code-app#496]
h1 Tag within the hero section on the landing page on mobile constantly resizes due to typing animation. Implement a fix to prevent the h1 from colliding with other adjacent elements.

**Comment by JVSONV** [comment]
I'd like to suggest that we add a slight black overlay on the background image to help boost the text color contrast ratio.

**Comment by jeromehardaway** [comment]
@JVSONV you solved this one already right?

**Comment by JVSONV** [comment]
Yes I believe it should be working correctly. I went back and saw the issues you stated after the initial fix. My second fix should rectify this issue now.
